### PR TITLE
Adding P1 steps to the GlobalSkylight automation

### PR DESCRIPTION
**Pull request by NeilGitGud**
Adding P1 steps to the GlobalSkylight automation

Added P1 tests to adjust Exposure value from min to max value, also loading a second set of High Contrast diffuse and specular images

Test run in Editor and passes:

Report:
[SUCCESS] Success: Global Skylight (IBL) Entity successfully created
[SUCCESS] Success: Entity has a Global Skylight (IBL) component
[SUCCESS] Success: UNDO Entity creation success
[SUCCESS] Success: REDO Entity creation success
[SUCCESS] Success: Entered game mode
[SUCCESS] Success: Exited game mode
[SUCCESS] Success: Entity is hidden
[SUCCESS] Success: Entity is visible
[SUCCESS] Success: Entity has the default Diffuse Image set
[SUCCESS] Success: Entity has the default Specular Image set
[SUCCESS] Success: Exposure set to maximum value
[SUCCESS] Success: Exposure set to maximum value
[SUCCESS] Success: Entity has the high contrast Diffuse Image set
[SUCCESS] Success: Entity has the high contrast Specular Image set
[SUCCESS] Success: Exposure set to maximum value
[SUCCESS] Success: Exposure set to maximum value
[SUCCESS] Success: Entity deleted
[SUCCESS] Success: UNDO deletion success
[SUCCESS] Success: REDO

Ran "TestSuite_Main.py" locally and all tests pass, but I did have to set "enable_prefab_system=True" on line 207:

AutomatedTesting\Gem\PythonTests\Atom\TestSuite_Main.py ..............................                           [100%]

================================================== warnings summary ===================================================
tools\lytesttools\ly_test_tools\o3de\editor_test.py:330
  c:\o3de_git\o3de\tools\lytesttools\ly_test_tools\o3de\editor_test.py:330: PytestCollectionWarning: cannot collect test class 'TestData' because it has a __init__ constructor (from: AutomatedTesting/Gem/PythonTests/Atom/TestSuite_Main.py::TestAutomation)
    class TestData:

AutomatedTesting/Gem/PythonTests/Atom/TestSuite_Main.py::TestMaterialEditorBasicTests::test_MaterialEditorBasicTests[windows-MaterialEditor-windows_generic-AutomatedTesting]
  c:\o3de_git\o3de\tools\lytesttools\ly_test_tools\_internal\pytest_plugin\test_tools_fixtures.py:296: DeprecationWarning: This method is deprecated and will be removed on 3/31/22. Please use another helper method instead.
    return ly_test_tools.launchers.launcher_helper.create_generic_launcher(workspace, launcher_platform, exe_file_name)

-- Docs: https://docs.pytest.org/en/latest/warnings.html
===================================== 30 passed, 2 warnings in 157.30s (0:02:37) ======================================

Signed-off-by: Neil Widmaier <nwidmaie@amazon.com>

### CODE REVIEW

File: AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponents_GlobalSkylightIBLAdded.py [hunk 1]
```diff
@@ -150,41 +162,91 @@ def AtomEditorComponents_GlobalSkylightIBL_AddedToEntity():
...
             specular_image_asset.id == global_skylight_component.get_component_property_value(
                                            AtomComponentProperties.global_skylight('Specular Image')))
 
-        # 10. Delete Global Skylight (IBL) entity.
+        #11. Set Exposure valuye to min value
+        global_skylight_component.set_component_property_value(
+            AtomComponentProperties.global_skylight('Exposure'), value=-5)
+        current_exposure_value = global_skylight_component.get_component_property_value(
+            AtomComponentProperties.global_skylight('Exposure'))
+        Report.result(Tests.exposure_set_max_value, current_exposure_value == -5)
```
Comment (smurly) [inline AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponents_GlobalSkylightIBLAdded.py:190]: should this be `Tests.exposure_set_min_value`?

File: AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponents_GlobalSkylightIBLAdded.py [hunk 2]
```diff
@@ -150,41 +162,91 @@ def AtomEditorComponents_GlobalSkylightIBL_AddedToEntity():
...
+            Tests.high_contrast_specular_image_set,
+            specular_image_asset.id == global_skylight_component.get_component_property_value(
+                                           AtomComponentProperties.global_skylight('Specular Image')))
+
+        #11. Set Exposure valuye to min value
+        global_skylight_component.set_component_property_value(
+            AtomComponentProperties.global_skylight('Exposure'), value=-5)
+        current_exposure_value = global_skylight_component.get_component_property_value(
+            AtomComponentProperties.global_skylight('Exposure'))
+        Report.result(Tests.exposure_set_max_value, current_exposure_value == -5)
```
Comment (smurly) [inline AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponents_GlobalSkylightIBLAdded.py:226]: again `Tests.exposure_set_min_value`

File: AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponents_GlobalSkylightIBLAdded.py [hunk 3]
```diff
@@ -150,41 +162,91 @@ def AtomEditorComponents_GlobalSkylightIBL_AddedToEntity():
...
+
+        #12. Set Exposure value to max value
+        global_skylight_component.set_component_property_value(
+            AtomComponentProperties.global_skylight('Exposure'), value=5)
+        current_exposure_value = global_skylight_component.get_component_property_value(
+            AtomComponentProperties.global_skylight('Exposure'))
+        Report.result(Tests.exposure_set_max_value, current_exposure_value == 5)
+
+        # 15. Delete Global Skylight (IBL) entity.
```
Comment (smurly) [inline AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponents_GlobalSkylightIBLAdded.py:235]: step 12
step 15

**Review by smurly — CHANGES_REQUESTED** [review]
please move the enter/exit game mode to after you set all the properties
so around step 12 or so

File: AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponents_GlobalSkylightIBLAdded.py [hunk 4]
```diff
@@ -150,41 +162,91 @@ def AtomEditorComponents_GlobalSkylightIBL_AddedToEntity():
...
         global_skylight_component.set_component_property_value(
             AtomComponentProperties.global_skylight('Specular Image'), specular_image_asset.id)
         Report.result(
-            Tests.specular_image_set,
+            Tests.default_specular_image_set,
             specular_image_asset.id == global_skylight_component.get_component_property_value(
                                            AtomComponentProperties.global_skylight('Specular Image')))
 
-        # 10. Delete Global Skylight (IBL) entity.
+        #11. Set Exposure valuye to min value
```
Comment (jromnoa) [inline AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponents_GlobalSkylightIBLAdded.py:185]: nitpick here and throughout: There should be a space after `#` comment markers

Comment (jromnoa) [reply AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponents_GlobalSkylightIBLAdded.py:185]: also shouldn't this be `# 10.` not 11?

**Review by jromnoa — CHANGES_REQUESTED** [review]

File: AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponents_GlobalSkylightIBLAdded.py (hunk 2 above)
Comment (NeilGitGud) [reply AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponents_GlobalSkylightIBLAdded.py]: Fixed!

File: AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponents_GlobalSkylightIBLAdded.py (hunk 4 above)
Comment (NeilGitGud) [reply AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponents_GlobalSkylightIBLAdded.py]: Fixed!

File: AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponents_GlobalSkylightIBLAdded.py (hunk 3 above)
Comment (NeilGitGud) [reply AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponents_GlobalSkylightIBLAdded.py]: Fixed!

File: AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponents_GlobalSkylightIBLAdded.py (hunk 1 above)
Comment (NeilGitGud) [reply AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponents_GlobalSkylightIBLAdded.py]: Fixed!

**Review by jromnoa — APPROVED** [review]

**Review by smurly — APPROVED** [review]
something is odd with the merge of changes for atom_constants.py
make sure you brought in development in your merge correctly

**Pull request merged**